Misc/NEWS.d/next/Core and Builtins/2023-05-25-21-40-39.gh-issue-104955.LZx7jf.rst

```diff
@@ -0,0 +1,2 @@
+Fix signature for the new :meth:`__release_buffer__` slot. Patch by Jelle
```

Comment:
```suggestion
Fix signature for the new :meth:`~object.__release_buffer__` slot. Patch by Jelle
```

I don't know if this works, just guessing.

Reply:
Seems like it should given this:

```
Doc/reference/datamodel.rst:.. method:: object.__release_buffer__(self, buffer)
```